tests/nimfrompy.nim

```diff
@@ -165,3 +165,45 @@ proc setMyFieldFromTt(self: AnotherTestType, value: TestType) {.exportpy.} =
 
 proc getMyField(self: AnotherTestType): int {.exportpy.} =
   self.myIntField
+
+# Raising Defects
+
+proc assertFalse(): void {.exportpy} =
+    # AssertionDefect
+    doAssert false
+
+proc attributeError(): string {.exportpy} =
+    # FieldDefect
+    var obj: MyObj
+    obj.c
```

Comment:
Admittedly, I'm not convinced this makes 100% sense to map `FieldDefect` to an `AttributeError`, but I'm curious what others think